sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -48,9 +50,9 @@ case class UnidentifiedWork(
   language: Option[Language] = None,
   dimensions: Option[String] = None,
   items: List[UnidentifiedItem] = Nil,
-  version: Int,
   visible: Boolean = true,
-  ontologyType: String = "Work")
+  ontologyType: String = "Work",
+  redirect: Option[Identifiable[None.type]] = None)
```

Comment:
Obviously the None.type thing is not right. Unfortunately the IdentityState trait is designed to always wrap a value of type T and we need to rejig it to make it just wrap a sourceIdentifier field. I'm going to try and do that next but I want to see if things break with this in